### PR TITLE
Recover properties lost during conversion

### LINKED ISSUE #278: Recover missing properties during parsing/writing
When converting from xml to xml, the following properties get lost/changed:
in packages:
- [x] comment

in files:
- [x] attribution texts
- [x] in fileContributors: `file contributor` is converted to `['file contributor']`

### PR DESCRIPTION
**Pull request by armintaenzertng**
fixes #278

### CODE REVIEW

**Review by meretp — COMMENTED** [review]
Looks good to me, I have nothing to comment on.

File: spdx/parsers/jsonyamlxml.py [hunk 1]
```diff
@@ -885,20 +885,27 @@ def parse_file_attribution_text(self, file_attribution_texts):
...
                 self.value_error("FILE_ATTRIBUTION_TEXT", file_attribution_texts)
+        elif isinstance(file_attribution_texts, str):
+            try:
+                return self.builder.set_file_attribution_text(
+                    self.document, file_attribution_texts
+                )
+            except CardinalityError:
+                self.more_than_one_error("FILE_ATTRIBUTION_TEXT")
+            except OrderError:
+                self.order_error("FILE_ATTRIBUTION_TEXT", "FILE_NAME")
```
Comment (nicoweidner) [inline spdx/parsers/jsonyamlxml.py:908]: I couldn't find a spot where these errors are raised. If you can't find one either, we can just remove the try/catch blocks

**Review by nicoweidner — APPROVED** [review]
Looks like there are some redundant try/catch blocks. I will leave it up to you whether you want to fix those in this PR

File: spdx/parsers/jsonyamlxml.py (hunk 1 above)
Comment (armintaenzertng) [reply spdx/parsers/jsonyamlxml.py:910]: I think we will leave it to the larger refactoring

**Pull request merged**